system.prop

```diff
@@ -86,11 +78,6 @@ ro.opengles.version=196610
 vidc.debug.perf.mode=2
 vidc.enc.dcvs.extra-buff-count=2
 
-# Camera
-persist.camera.cpp.duplication=false
-camera.disable_zsl_mode=1
-media.camera.ts.monotonic=0
```

Comment:
**Edit:** didn't notice, that the props are defined above, so we can keep these changes.